src/main/java/cz/fi/muni/pa165/esports/enums/Gender.java

```diff
@@ -0,0 +1,10 @@
+package cz.fi.muni.pa165.esports.enums;
+
+/**
+ * @author Radovan Tomasik
+ */
+public enum Gender {
+    MALE,
+    FEMALE,
+    SOMETHINGELSE
```

Comment:
Maybe it would be more politically/socially correct to call the third option `OTHER` instead of `SOMETHINGELSE` :smile: 
But hey, it's your code, I don't insist on the change. :upside_down_face:

Reply:
Done :D